config/nginx.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen       ${NGINX_PORT};
+    listen  [::]:${NGINX_PORT};
+    server_name  ${NGINX_HOST};
+
+    #access_log  /var/log/nginx/host.access.log  main;
```

Comment:
Would this be helpful to enable by default?

Reply:
Potentially yes.  However, if someone is not familiar with Docker, it would be a bit of a process for them to actually see that log unless we mount the log folder from outside of the container.